src/Playwright/Transport/Converters/EvaluateArgumentValueConverter.cs

```diff
@@ -252,11 +252,8 @@ private static object ToExpectedType(object parsed, Type t, IDictionary<object,
 
             foreach (var kv in parsedExpando)
             {
-                var property = t.GetProperties().FirstOrDefault(prop => string.Equals(prop.Name, kv.Key, StringComparison.OrdinalIgnoreCase));
-                if (property != null)
-                {
-                    property.SetValue(objResult, ToExpectedType(kv.Value, property.PropertyType, visited));
-                }
+                var property = Array.Find(t.GetProperties(), prop => string.Equals(prop.Name, kv.Key, StringComparison.OrdinalIgnoreCase));
+                property?.SetValue(objResult, ToExpectedType(kv.Value, property.PropertyType, visited));
```

Comment:
What is the minimal .Net version that supports  `?.` operator, are we confident that we'll not need to build with it?

Reply:
.NET takes care of that via the TFM, we can use any newer C# language feature how we like.